docs/pages/review/overview.mdx

```diff
@@ -0,0 +1,83 @@
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { CODE } from '~/ui/components/Text';
+import { BuildIcon, LayersTwo02Icon, GithubIcon } from '@expo/styleguide-icons';
+
+This page outlines different strategies to share a preview version of your app using internal distribution and EAS Update. Sharing a preview version with testers allows them to download and run it on their devices as if it were distributed through a store.
+
+## Internal distribution
+
+Internal distribution is a strategy for sharing an app with a limited audience. It is used for testing purposes within a team or organization, or a selected group of testers, or by an individual developer that is testing their own app. Android and iOS provide different ways to install a build of your application directly on a device.
+
+Google Play beta provides both closed and open tracks. You can invite specific users to test your app in the closed beta. In the open beta, you can make your app available to anyone participating in the beta program. You could alternatively distribute the APK directly to testers.
+
+Apple offers ad hoc provisioning to distribute your app to test on devices registered to your Apple Developer account. You can also use TestFlight to share your app with up to 10,000 external testers by [uploading your app](/submit/ios/).
+
+EAS Build makes it easy to generate a shareable URL that your team can use to download an installable APK for Android or an ad hoc provisioned app for iOS.
+
+<BoxLink
+  title="How to set up an internal distribution build"
+  description="Learn how EAS Build provides shareable URLs for your builds with your team for internal distribution."
+  href="/build/internal-distribution/"
+  Icon={BuildIcon}
+/>
+
+## Using EAS Update with development builds
+
+You can use [development builds](/develop/development-builds/introduction/) to load previews of your app during the review stage by publishing the preview with EAS Update.
+
+<Collapsible summary="Can I use Expo Go for reviewing releases?">
+
+Even though Expo Go is an open-source sandbox that can be good for previewing isolated prototypes on Android and iOS, it is not intended for production apps. It should be avoided during the preview process of your app.
+
+</Collapsible>
+
+<BoxLink
+  title="Get started with EAS Update"
+  description={
+    <>
+      Learn how to get started using <CODE>expo-updates</CODE> library and use EAS Update in your
+      project.
+    </>
+  }
+  href="/eas-update/getting-started/"
+  Icon={LayersTwo02Icon}
+/>
+
+<BoxLink
+  title="Develop with EAS Update"
+  description="Learn how to use EAS Update to push changes to iterate faster with your team. This process allows any team member or reviewer with a registered device to review the latest version of your app by scanning the QR code."
+  href="/eas-update/develop-faster/"
+  Icon={LayersTwo02Icon}
+/>
```

Comment:
this link may not be particularly useful anymore after https://github.com/expo/expo/pull/27461 - wdyt @wschurman / @amandeepmittal?

Reply:
I just saw Will's PR and got the context. Will remove this link.

Reply:
Done ✅